lib/commands/connect/regions.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const cli = require('heroku-cli-util');
+
+const prodRegions = ['us', 'eu', 'tokyo', 'virginia', 'oregon', 'frankfurt', 'sydney'];
```

Comment:
I wasn't certain that we'd definitely be able to claim to be in every region that Heroku supports. If we tell people to view `heroku regions` then that would mean that if something new is launched we'd have to be already deployed there (which I am not certain is the case). I just wanted to prevent users from shooting themselves in the feet and getting cryptic errors